src/legacy/config/config-set.ts

```diff
@@ -248,6 +248,15 @@ export class ConfigSet {
    * @internal
    */
   private _setupConfigSet(options: TsJestTransformOptions['transformerConfig']): void {
+    // tsconfig
```

Comment:
Moved this block up to make `parsedTsConfig` available at diagnostic handling time